### PR TITLE
PLAT-116038: Support `deactivated` audio guidance in components

### PR DESCRIPTION
**Pull request by ybsung**
Read out `deactivated` audio guidance properly in components.

There are some reasons why `deactivated` was not read out.
One for not supporting `disabled` status, another for not reading out `deactivated`, the other for no `disabled` prop in the samples.

ColorPicker, ImageItem, Slider, Picker
- Support `disabled`
- Readout `deactivated` when `disabled`

DateTimePicker
- Support `disabled`

Input
- Readout `deactivated` when `disabled`

[Additional consideration] ButtonView in qa-a11y
- Fix for the wrong `aria-label`

To verify A11y disabled features, please use the qa-a11y sample.

Enact-DCO-1.0-Signed-off-by: YB Sung (yb.sung@lge.com)

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]
:lgtm:

File: ColorPicker/ColorPicker.js [hunk 1]
```diff
@@ -80,6 +80,14 @@ const ColorPickerBase = kind({
 		 */
 		direction: PropTypes.string,
 
+		/**
+		 * Disables the `ColorPicker`.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		disabled: PropTypes.bool,
```
Comment (MikyungKim) [inline ColorPicker/ColorPicker.js:89]: Should this be on the knob of sampler?

File: DateTimePicker/DateTimePicker.js [hunk 2]
```diff
@@ -31,23 +32,33 @@ import css from './DateTimePicker.module.less';
 const DateTimePickerBase = kind({
 	name: 'DateTimePickerBase',
 
+	propTypes: /** @lends agate/DateTimePicker.DateTimePicker.prototype */ {
+		/**
+		 * Disables the `ImageItem`.
```
Comment (MikyungKim) [inline DateTimePicker/DateTimePicker.js:37]: `DateTimePicker`?

File: samples/qa-a11y/src/views/Scroller.js [hunk 3]
```diff
@@ -33,6 +42,7 @@ const ScrollerView = () => {
 				</CheckboxItem>
 			</Header>
 			<Scroller
+				disabled={disabled}
```
Comment (MikyungKim) [inline samples/qa-a11y/src/views/Scroller.js:45]: Is this work?

**Comment by MikyungKim** [comment]
I left some comments. Should `disabled` work for Scroller and VL too? I think it's not possible.

**Pull request merged**